parser/src/main/java/com/microsoft/gctoolkit/parser/GCLogParser.java

```diff
@@ -52,18 +58,37 @@ public void setClock(DateTimeStamp newValue) {
     protected final void advanceClock(DateTimeStamp now) {
         if (now == null)
             return;
-            // now can be the same but it can't be less than
+        // now can be the same but it can't be less than
         else if (now.before(getClock())) {
-            if ( ! now.equals(getClock()))
-                LOGGER.log(Level.WARNING, "Thread: {0}, abort GC log parsing. Time traveled backwards from {1} to {2}", new Object[]{Thread.currentThread().getName(), getClock().toString(), now.toString()});
+            LOGGER.log(Level.WARNING, "Thread: {0}, abort GC log parsing. Time traveled backwards from {1} to {2}", new Object[]{Thread.currentThread().getName(), getClock().toString(), now.toString()});
```

Comment:
This doesn't actually abort GC log parsing.

Reply:
It doesn't and that is by design.

Reply:
Then the log message needs to be changed since it says "abort GC log parsing".

Reply:
oops, I don't recall writing an abort message. I will fix it